__tests__/cmds/versions.test.js

```diff
@@ -37,6 +37,8 @@ jest.mock('../../src/lib/prompts');
 describe('rdme versions*', () => {
   beforeAll(() => nock.disableNetConnect());
 
+  beforeEach(() => jest.resetModules());
```

Comment:
Because we're mocking out `process.env.CIRCLECI` to test the [ci-info](https://npm.im/ci-info) package if we don't reset modules here then that package will get loaded within our `src/lib/enquirer.js` wrapper without the env variable, and cached -- so when the test that does need it, and does mock that environment variable out, the cached module doesn't see it.

Reply:
I feel like it might make more sense to do this in a global `beforeEach` call at some point